### PR TITLE
Add IntersectionObserver v1

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

**Comment by foolip** [comment]
@szager if you have any thoughts on how/whether to split IntersectionObserver as high-level features, please let me know.

**Comment by foolip** [comment]
I've split v2 into https://github.com/web-platform-dx/web-features/pull/628 so that v1 can be merged without figuring out the spec URL questions for v2.

File: feature-group-definitions/intersection-observer.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: IntersectionObserver
```
Comment (ddbeck) [inline feature-group-definitions/intersection-observer.yml:1]: I think there's a case to be made for either here, but the spec uses the phrase "intersection observer" (not just the interface name) pretty routinely. I think it's more readable with the space and we should do the same. But caniuse does use the interface name, so this is not a blocking suggestion

```suggestion
name: Intersection observer
```

**Review by ddbeck — APPROVED** [review]

File: feature-group-definitions/intersection-observer.yml (hunk 1 above)
Comment (foolip) [reply feature-group-definitions/intersection-observer.yml:1]: I think we should use the phrase form when there's no ambiguity and there's some precedent for it, so I'll accept your suggestion.

**Pull request merged**